plugin_interpreter/src/rethink_interface.py

```diff
@@ -186,58 +161,14 @@ def check_for_plugin(self, plugin_name):
         """
         try:
             rethinkdb.db("Plugins").table(plugin_name).run(
-                self.validate_db(self.rethink_connection)
+                connect(host=self.host, port=self.port)
             )
             return True
         except rethinkdb.ReqlOpFailedError:
             return False
 
-    @staticmethod
-    def validate_db(connection):
```

Comment:
suggest deprecating rather than outright removing for posterity.

validate_db could remain a staticmethod returning validate_brain(connection)

https://github.com/ramrod-project/database-brain/blob/2b09a2d9785c0addf942125c905539d39f361294/schema/brain/connection.py#L110

pragma nocover it because cc is rude